src/test/java/uk/gov/pay/directdebit/payments/api/TransactionRequestValidatorTest.java

```diff
@@ -68,4 +70,19 @@ public void shouldThrowIfFieldsAreInvalid() {
         thrown.reportMissingExceptionWithMessage("InvalidFieldsException expected");
         transactionRequestValidator.validate(request);
     }
+
+    @Test
+    public void shouldThrowInvalidFieldsExceptionIfAmountFieldIsNonNumeric() {
+        Map<String, String> request = ImmutableMap.of(
+                "amount", "eqweqw",
+                "description", "bla",
+                "return_url", "bla",
```

Comment:
As discussed with @alexbishop1 probably it's good to have valid URL, so that test actually tests invalid `amount` only?

Reply:
cool